cli/csstools-cli/.nvmrc

```diff
@@ -0,0 +1 @@
+v16.13.1
```

Comment:
Slightly nit and potentially unrelated but should we just add 16?

Reply:
16 sounds good.

We can change it in a different PR, or straight to main.